commands/commandlog-get.md

```diff
@@ -0,0 +1,33 @@
+The `COMMANDLOG GET <count> <type>` command returns entries from the command log of specified type in chronological order.
```

Comment:
The syntax with parameters will be rendered just above this text from the command JSON file, so let's not include the syntax here. It will just look like duplicated information.

```suggestion
The `COMMANDLOG GET` command returns entries from the command log of specified type in chronological order.
```